src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -243,7 +243,7 @@ class BlockchainLMDB : public BlockchainDB
   virtual uint64_t get_num_outputs(const uint64_t& amount) const;
 
   virtual output_data_t get_output_key(const uint64_t& amount, const uint64_t& index);
-  virtual void get_output_key(const uint64_t &amount, const std::vector<uint64_t> &offsets, std::vector<output_data_t> &outputs, bool allow_partial = false);
+  virtual void get_output_key(const std::vector<uint64_t> &amounts, const std::vector<uint64_t> &offsets, std::vector<output_data_t> &outputs, bool allow_partial = false);
```

Comment:
If `amounts` were a `epee::span<const uint64_t>` then the caller wouldn't have to allocate a vector for the one element case - a one element array would suffice.

Should the one element behavior of `amounts` be documented in the header now?

Reply:
Yes. It is.